usaspending_api/database_scripts/job_archive/management/commands/generate_cares_act_test_data.py

```diff
@@ -0,0 +1,297 @@
+"""
+Jira Ticket Number(s): DEV-5343
+
+    Generate test CARES Act monthly submissions and DEFC Files B and C data.
+
+Expected CLI:
+
+    $ ./manage.py generate_cares_act_test_data 2020 2  # which are FY and FQ - use any FY and FQ you want
+
+Purpose:
+
+    This script deterministically generates sample CARES Act data for testing and development
+    purposes.  It generates this data from existing data by duplicating and modifying existing
+    submissions and File A/B/C records.  Data points are adjusted in an attempt to make them
+    seem realistic and true to their actual source submissions.
+
+    These data will not be perfect, obviously, but they should be sufficient for testing.
+
+Life expectancy:
+
+    This file should live until CARES Act features have gone live.  Also delete the
+    generate_cares_act_test_data_sqls directory and supporting SQL files.
+
+"""
+import logging
+import re
+
+from argparse import ArgumentTypeError
+from collections import namedtuple
+from datetime import timedelta, date
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from pathlib import Path
+from usaspending_api.common.helpers.sql_helpers import execute_sql_return_single_value, execute_dml_sql
+from usaspending_api.common.helpers.timing_helpers import ScriptTimer
+from usaspending_api.references.models import DisasterEmergencyFundCode
+from usaspending_api.submissions.models import SubmissionAttributes
+
+
+logger = logging.getLogger("script")
+
+
+class OneLineTimer(ScriptTimer):
+    def log_starting_message(self):
+        pass
+
+    def log_success_message(self):
+        pass
+
+
+class Command(BaseCommand):
+    help = (
+        "This script generates test CARES Act submissions and File A/B/C data from existing "
+        "pre-CARES Act submissions."
+    )
+
+    fiscal_year = None
+    fiscal_quarter = None
+    allow_rds = False
+    vacuum = False
+
+    fiscal_period = None
+    clone_periods = None
+    period_ratios = None
+    period_starts = None
+    period_ends = None
+
+    def add_arguments(self, parser):
+
+        parser.add_argument(
+            "fiscal year", type=self.validate_fiscal_year, help='Fiscal year to be "enhanced".',
+        )
+
+        parser.add_argument(
+            "fiscal quarter", type=self.validate_fiscal_quarter, help='Fiscal quarter to be "enhanced".',
```

Comment:
Is there a reason we're doing this by quarter instead of by period? Our first data will be coming in as period 7.

Reply:
It converts a quarter into periods by cloning quarter submissions across the one to two other periods for the quarter.  I assume if we're receiving monthly data for period 7 we don't need to convert it?